packages/server/src/utils.ts

```
@@ -613,35 +613,79 @@ export function ensureDisposableStackRegisteredForTerminateEvents(
+    const nodeEvents: typeof import('node:events') =
+      globalThis.process?.getBuiltinModule?.('node:events');
+    // @ts-expect-error - We know kMaxEventTargetListeners is available in node:events
+    if (nodeEvents?.kMaxEventTargetListeners) {
+      // @ts-expect-error - See https://github.com/nodejs/node/pull/55816/files#diff-03bd4f07a1006cb0daaddced702858751b20f5ab7681cb0719c1b1d80d6ca05cR31
+      this[nodeEvents.kMaxEventTargetListeners] = 0;
+    }
+  }
 
   throwIfAborted(): void {
+    if (this._nativeCtrl?.signal?.throwIfAborted) {
+      return this._nativeCtrl.signal.throwIfAborted();
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)